src/main/resources/validationrules/iac/iac-judicial-org-role-mapping.drl

```diff
@@ -258,3 +259,37 @@ then
       .build());
       logMsg("Rule : iac_fee_paid_judge_org_role");
 end;
+
+/*
+ * IAC "judge" org role can be created by any existing judicial office holder having
+ * "IAC Tribunal Judge (Fee-Paid)" business role.
+ */
+rule "iac_judge_org_role_for_fee_paid_judge"
+when
+  $f:  FeatureFlag(status && flagName == FeatureFlagEnum.IAC_JRD_1_0.getValue())
+  $joh: JudicialOfficeHolder(office in ("IAC Tribunal Judge (Fee-Paid)"))
+  $bk: JudicialBooking(userId == $joh.userId)
+then
+   Map<String,JsonNode> attribute = new HashMap<>();
+   attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("IA"));
+   attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($bk.getLocationId()));
```

Comment:
I think, getLocationId() can throw NPE as this filed can be null as per the create booking API.